packages/paste-website/src/pages/articles/2021-04-29-insights-&-metrics-that-inform-the-paste-design-system.mdx

```diff
@@ -0,0 +1,280 @@
+---
+title: Insights & Metrics that Inform the Paste Design System
+description: Learn how the Twilio Paste design system captures data to gain insights and track metrics which guide the development of the system.
+slug: /articles/2021-04-29-insights-&-metrics-that-inform-the-paste-design-system/
```

Comment:
Minor thing: should we encode the ampersand here, or make it "and"?

Reply:
Probably easier to make it `and`